src/cli/README.md

```diff
@@ -421,6 +422,35 @@ Invalidate a prefix in the Network Data.
 Done
 ```
 
+### router list
+
+List allocated Router IDs
+
+```bash
+> router list
+8 24 31
+Done
+```
+
+### router \<id\>
+
+Print diagnostic information for a Thread Router.  The `id` may be a Router ID or an RLOC16.
+
+```bash
+> router 38
```

Comment:
'router 38' might be an un-alloc example since it is not listed in 'router list';
we might also want to give an example of allocated RLOC16? since 'rloc16' cmd returns RLOC16 without '0x' prefix while 'router id' accepts RLOC16 with '0x' prefix.
`> rloc16
5c00
Done
> router 5c00
Error 6
> router 0x5c00
Alloc: 1
Router ID: 23
Rloc: 5c00
Next Hop: 5c00
Link: 0
Done
`